nova/compute/resource_tracker.py

```diff
@@ -1685,6 +1688,7 @@ def _remove_deleted_instances_allocations(self, context, cn,
                             "the source host that might need to be removed: "
                             "%s.",
                             instance_uuid, instance.host, instance.node, alloc)
+        METRICS_HANDLER.gauge("allocation.orphans", orphan_allocations)
```

Comment:
This would then be `metrics.METRICS_HANDLER.gauge`.
I would prefer to have it right as `metris.gauge`. I don't think everything needs to be an object.

Reply:
good idea, code is much shorter now.